client/my-sites/stats/stats-redirect/index.tsx

```diff
@@ -73,8 +70,7 @@ const StatsRedirectFlow: React.FC< StatsRedirectFlowProps > = ( { children } ) =
 	}, [ dispatch, siteId, isLoadingNotices, purchaseNotPostponed ] );
 
 	// Render conditions (for readability).
-	const shouldRenderPaywall =
-		! isLoading && ! skipPaywallFlow && redirectToPurchase && siteSlug && canUserManageOptions;
+	const shouldRenderPaywall = ! isLoading && redirectToPurchase && siteSlug && canUserManageOptions;
```

Comment:
We might still want to guard the rest of the pages? @dognose24

Reply:
Yes, it would be better to gate pages like the `Summary` page. By the way, it would be great to reuse `shouldGateStats`, though. 🤔